src/quantforge/backtest_strategy.py

```diff
@@ -0,0 +1,597 @@
+"""
+Backtest a strategy.
```

Comment:
This file should be in the back test folder. There is already a folder for this.